bittensor/core/extrinsics/registration.py

```diff
@@ -399,11 +379,6 @@ def burned_register_extrinsic(
         logging.info(f"\t\tcoldkey: <blue>{neuron.coldkey}</blue>")
         return True
 
-    if prompt:
-        # Prompt user for confirmation.
-        if not Confirm.ask(f"Recycle {recycle_amount} to register on subnet:{netuid}?"):
-            return False
-
```

Comment:
Do you think it will be valuable to just log this information for the user? 
I think recycle_amount would be great to show along with subnet

Reply:
done